behat.php.dist

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
Should be removed
https://github.com/Behat/Behat/pull/1537#discussion_r1848411572

Reply:
done